src/Akka.Persistence.MongoDb/Journal/MongoDbJournal.cs

```diff
@@ -121,9 +129,13 @@ protected override void PreStart()
 
                 if (_settings.AutoInitialize)
                 {
-                    collection.Indexes.CreateOneAsync(
-                        Builders<MetadataEntry>.IndexKeys
-                            .Ascending(entry => entry.PersistenceId))
+                    var modelWithAscendingPersistenceId = new CreateIndexModel<MetadataEntry>(
+                        Builders<MetadataEntry>
+                            .IndexKeys
+                            .Ascending(entry => entry.PersistenceId));
+
+                    collection.Indexes
+                        .CreateOneAsync(modelWithAscendingPersistenceId, cancellationToken:CancellationToken.None)
```

Comment:
ℹ️ I'm not sure `cancellationToken` is necessary here; did it by instinct. Happy to remove since it likely defaults to that anyway. Your call. 👍

Reply:
It's fine by me this looks great. Thank you.